.vscode/tasks.json

```diff
@@ -0,0 +1,12 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
+    // for the documentation about the tasks.json format
+    "version": "0.1.0",
```

Comment:
My mistake, i hate that ctrl+enter commits everything even if it wasn't staged. I keep doing it by accident.